src/Utils/Json.php

```diff
@@ -89,7 +89,7 @@ public static function decode($json, $options = 0)
 		}
 		$value = call_user_func_array('json_decode', $args);
 
-		if ($value === NULL && $json !== '' && strcasecmp($json, 'null')) { // '' is not clearing json_last_error
+		if ($value === NULL && $json !== '' && strcasecmp(trim($json, " \t\n\r"), 'null') != 0) { // '' is not clearing json_last_error
```

Comment:
I considered using `preg_match()`, but I decided using `trim()` would be better, because it is faster for the valid case (i.e., when the decoded JSON was a correct `"null"`), while it is slightly slower in the erroneous case. I believe this is the correct decision.